qcodes/tests/drivers/test_ami430.py

```diff
@@ -494,6 +495,170 @@ def test_ramp_rate_exception(current_driver):
         ix.ramp_rate(target_ramp_rate)
 
 
+def test_simultaneous_ramp_mode_does_not_reset_individual_axis_ramp_rates_if_nonblocking_ramp(
+    current_driver, caplog, request
```

Comment:
Could do the usual rename fixture and set name dance to avoid this warning (for current_driver)

Reply:
yes